### ISSUE TITLE
Fix docker-compose

### ISSUE THREAD

**Issue by alexandresaura** [Codeminer42/Punchclock#108]
There are some bugs in the docker-compose method to run the application.
First that I noticed was that the nodejs version on the docker is 16.13.1 and we need the version 15.14.0 to run the app.
Another bug is that we need python 2 if we wanna use yarn to install the dependencies (check if we could change to npm or we will install python on container).
And it could have more bugs.

**Comment by wenderjean** [comment]
I'll take a look what is the best approach to solve this issue.

**Comment by geeksilva97** [comment]
Could it be the `ENV NODE_VERSION` from the `codeminer42/ci-ruby:2.7` image?

https://hub.docker.com/layers/ci-ruby/codeminer42/ci-ruby/2.7/images/sha256-85ff64f4d17ae1433e36c10cdeadf7ee1e511effa13110bc102122b080f9f5d4?context=explore

**Comment by geeksilva97** [comment]
It is certainly related to the Node version. Precisely it involves the ´node-sass´ which is a `@rails/webpacker` dependency. I did some experiments and I potentially found the problem and the solution. Let me try a little bit more on this.

**Comment by geeksilva97** [comment]
As the image used Node 16 I tried to update the engine at `package.json`. So a problem involving `node-sass` started happening. After some researching and experiments i found that `node-sass` is one of `@rails/webpacker` dependencies. Moreover, I found found that the `node-sass`'s version what was being installed was not compatible with Node 16.

So I did more tests until reach the solution that is updating the `@rails/webpacker`.
